docs/nodes/NODES.md

```diff
@@ -0,0 +1,68 @@
+# Using the Node Editor
+
+The nodes editor is a blank canvas allowing for the use of individual functions and image transformations to control the image generation workflow. The node processing flow is usually done from left (inputs) to right (outputs), but is not always the case the more complex the node graph becomes. Nodes inputs and outputs are connected by dragging connectors from node to node.
```

Comment:
Easier to say that each node’s inputs are on the left and its outputs are on the right. There’s no other particular reason that the graph goes from left to right.

Reply:
You might want to mention that inputs and outputs are color coded to indicate which ones make valid connections.